### PR TITLE
Add Screenshot and Change README.md

### PR DESCRIPTION
**Pull request by localanu**

### CODE REVIEW

**Comment by janus2** [comment]
With the application Screenshot you can take the screenshot of the window without the background.  It looks more professional. (You can find the application under /boot/systems/apps/Screenshot)

**Comment by localanu** [comment]
Should I close and make pull req again?

**Comment by localanu** [comment]
I updated it

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+![FontBoy screenshot](FontBoy.png "FontBoy")
```
Comment (humdingerb) [inline README.md]: It'd look better after the introduction.
While you're at it, remove the "Version 0.9.7". Future release may not need a changed readme.

Comment (localanu) [reply README.md]: I updated it

**Comment by humdingerb** [comment]
Thanks!

**Comment by localanu** [comment]
Your wellcome

**Pull request merged**